src/Features/CSharp/Portable/MakeTypeAbstract/CSharpMakeTypeAbstractCodeFixProvider.cs

```diff
@@ -53,15 +50,13 @@ protected override bool IsValidRefactoringContext(SyntaxNode? node, [NotNullWhen
                     return false;
             }
 
-            var enclosingType = node.FirstAncestorOrSelf<TypeDeclarationSyntax>();
-            if (!enclosingType.IsKind(SyntaxKind.ClassDeclaration))
+            typeDeclaration = node.FirstAncestorOrSelf<TypeDeclarationSyntax>();
```

Comment:
don't do this.  you will assign the value, even when returning false.  it would be far cleaner to keep the out-value 'null' when 'false' is returned.